### DISCUSSION TITLE
New Source Suggestion

### DISCUSSION THREAD

**Discussion by ManojLingala** [dailydotdev/daily#829] [General]
Hey , 

I just joined today . As a fan of blockchain and crypto, you may have noticed that there is not much content related to this topic on our platform. 

So , I have suggested the below:

1. Cointelegraph RSS feed: https://cointelegraph.com/rss
2. Blockchain News RSS feed: https://www.the-blockchain.com/feed/
3. Decrypt RSS feed: https://decrypt.co/feed
4. Blockonomi RSS feed: https://blockonomi.com/feed/

I was wondering , is there any prompt to notify user to in case if  these feeds are already available which reduces the duplication and review process 🤔. I believe this would be a simple check in the backend

**Comment by nimrodkra** [comment]
@ManojLingala, thank you very much for the contribution! You would get an automated notification if the sources already existed in our system. If the suggestion passes through, it means that these sources are not part of the feed at the moment. So in that sense, the next step would be for our team to review it. Once the review is done, you will get an email notification with the result.
